change/@uifabric-charting-2020-07-10-10-58-14-user-v-gorraj-TSK_4283359.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Can you also ensure you update the legends, even that has to be selected. I am sending you the figma.

Reply:
I have updated as per figma